test/GenerateOgmiosFixtures.purs

```diff
@@ -0,0 +1,148 @@
+module Test.GenerateOgmiosFixtures
```

Comment:
I think this should be `Test.Ogmios.GenerateFixtures` and this file and its FFI module should be moved accordingly